src/test/test-context-providers.tsx

```diff
@@ -6,8 +6,12 @@ import * as FeatureToggle from '../ducks/feature-toggles';
 import * as Egenvurdering from '../ducks/egenvurdering';
 import * as Oppfolging from '../ducks/oppfolging';
 import * as UlesteDialoger from '../ducks/ulestedialoger';
+import * as Jobbsokerbesvarelse from '../ducks/jobbsokerbesvarelse';
 import * as React from 'react';
-import { DeepPartial } from 'redux';
+
+type DeepPartial<T> = {
```

Comment:
brrukes ikke denne 2 steder nå? trekke ut i egen, eller for tidlig?

Reply:
Jørgen sin PR (#511) fjerner DeepPartial fra `banner.test.tsx`, og da vil vi bare trenge den her i `test-context-providers` 😃